modules/ocf/manifests/firewall/allow_http.pp

```diff
@@ -0,0 +1,20 @@
+# firewall input rule to allow http and https
+class ocf::firewall::allow_http {
+  ocf::firewall::firewall46{
+    '101 allow https':
+      opts => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => 'https',
+        action => 'accept',
+      };
+
+    '102 allow http':
```

Comment:
What is this sequentially numbered naming convention?

Reply:
i don't think we need to sequentially number except in cases where order matters -- these can both just be 100, can't they?

Reply:
Do they need a numeric prefix if order doesn't matter?

Reply:
yes because there are rules at the start/end (often global ones) which need to be there

there are groups like these though where you can flip the order, they might as well just have the same prefix